Marlin/src/inc/Conditionals_LCD.h

```diff
@@ -1141,6 +1141,7 @@
  *  - GRAPHICAL_TFT_UPSCALE
  */
 #if ENABLED(MKS_TS35_V2_0)          // Most common: ST7796
+  #define TFT_DEFAULT_DRIVER 0x7796 // ST7796
```

Comment:
You can use the driver define ST7796

Reply:
no, it is not available error: 'ST7796' was not declared in this scope
and I didn't want to include Marlin/src/lcd/tft_io/tft_io.h for 1 definition

Reply:
I already tested  #define TFT_DRIVER ST7796 and it works.